org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java

```diff
@@ -1598,7 +1604,7 @@ private String stripHtml(String htmlString, boolean all) {
 	 */
 	private ReportEngine runReport(int AD_Process_ID, int AD_PrintFormat_ID, String parameters) {
    		MProcess process = MProcess.get(Env.getCtx(), AD_Process_ID);
-		if (!process.isReport() || process.getAD_ReportView_ID() == 0)
+		if (!process.isReport())
```

Comment:
why the need to remove the checking here ? Isn't JasperReport never reach here ?

Reply:
yup, this my first approach also